python/lsst/fgcmcal/fgcmBuildStars.py

```diff
@@ -306,17 +306,17 @@ def fgcmMakeAllStarObservations(self, groupedDataRefs, visitCat,
 
                     # This is the difference between the mag with background correction
                     # and the mag without background correction.
-                    tempCat[deltaMagBkgKey][:] = (-2.5*np.log10(sources[instFluxKey][goodSrc.selected] -
-                                                                localBackground[goodSrc.selected]) -
+                    tempCat[deltaMagBkgKey][:] = (-2.5*np.log10(sources[instFluxKey][goodSrc.selected]
+                                                                - localBackground[goodSrc.selected]) -
```

Comment:
Doesn't this still have a trailing `-` operator?

Reply:
Yes. It's weird. I think it might be a bug in flake8 not understanding how the `-` on this line and the `-` on the start of the next line are two different operators.